tests/integration_tests/test_skopt.py

```diff
@@ -154,8 +158,10 @@ def _objective(trial):
     p8 = trial.suggest_discrete_uniform("p8", 2.2, 2.2, 0.5)
     p9 = trial.suggest_categorical("p9", ["9", "3", "0", "8"])
     assert isinstance(p9, str)
+    p10 = trial.suggest_int("p10", 1, 8, log=True)
+    p11 = trial.suggest_int("p11", 3, 9, step=2, log=True)
```

Comment:
I think it would be better to put `suggest_int(log=True)` just below `suggest_int` to make it look better.

Reply:
Thanks! I updated it.